highcharts-api/highcharts/1-simple-column/main.js

```diff
@@ -0,0 +1,65 @@
+const generateRandomNumber = () => Math.floor(Math.random() * 10);
+const generateRandomTriplet = () => Array.from({ length: 3 }, generateRandomNumber);
+
+Highcharts.chart('container', {
+    chart: {
+        type: 'column',
+
+        events: {
+            load() {
+                const chart = this;
+                const maxY = chart.yAxis[0].dataMax;
```

Comment:
Repeating `const` is redundant, with many `const` variables you can use a comma:
```suggestion
const chart = this,
    maxY = chart.yAxis[0].dataMax;
```